spec/lib/dispatch-rider/demultiplexer_spec.rb

```diff
@@ -28,17 +28,17 @@ def process(options)
     thread
   end
 
-  let(:error_handler){ ->(message, exception){ raise exception }}
+  let(:error_handler) { ->(message, exception) { raise exception } }
```

Comment:
Unused block argument - `message`. If it's necessary, use `_` or `_message` as an argument name to indicate that it won't be used.